pyglimmpse/multirep.py

```diff
@@ -463,29 +464,30 @@ def wlk_two_moment_null_approx(rank_C: float,
         w = np.exp(np.sum(-np.log(np.ones((1, min_rank_C_U)) + eval_HINVE)))
 
     if min_rank_C_U == 1:
-        df2 = total_N - rank_X -rank_U + 1
+        df2 = total_N - rank_X - rank_U + 1
         rs = 1
         tempw = w
     else:
-        rm = total_N - rank_X - (rank_U - rank_C + 1)/2
-        rs = np.sqrt((rank_C*rank_C*rank_U*rank_U - 4) / (rank_C*rank_C + rank_U*rank_U - 5))
-        r1 = (rank_U * rank_C - 2)/4
+        rm = total_N - rank_X - (rank_U - rank_C + 1) / 2
+        rs = np.sqrt((rank_C * rank_C * rank_U * rank_U - 4) / (rank_C * rank_C + rank_U * rank_U - 5))
+        r1 = (rank_U * rank_C - 2) / 4
         if np.isnan(w):
             tempw = float('nan')
         else:
-            tempw = np.power(w, 1/rs)
+            tempw = np.power(w, 1 / rs)
         df2 = (rm * rs) - 2 * r1
 
     if np.isnan(tempw):
         omega = float('nan')
     else:
         omega = df2 * (1 - tempw) / tempw
 
+    warning_message = 'Power is missing because because the noncentrality could not be computed.'
```

Comment:
Same problem as before. defining a a warning that we might not use is confusing here.
define it if we hit that condition.